mayastor/src/core/env.rs

```diff
@@ -639,6 +640,22 @@ impl MayastorEnvironment {
         cfg.apply();
     }
 
+    fn get_service_config(&self) {
+        if self.mbus_endpoint.is_some() {
+            tokio::runtime::Runtime::new().unwrap().block_on(async {
+                let _reply = GetConfig::request(
+                    RequestConfig {
+                        kind: ConfigKind::MayastorConfig,
+                    },
+                    Channel::Kiiss,
+                    &subsys::message_bus(),
+                );
+                //.await;
+                //let _config = _reply.unwrap();
```

Comment:
comments should be removed?

Reply:
Yeah though I'm tempted to remove it altogether, that was just to leave as an example of how to call into the message bus

Reply:
I opted to leave the code in, but just not call the function so this way we don't need to comment out things that would fail without the service being deployed.